Tasks/DownloadGitHubNugetPackageV1/restorecommand.ts

```diff
@@ -230,21 +228,17 @@ function GetGitHubUser(endpointId: string): Promise<string> {
         token = externalAuth.parameters["accessToken"];
     }
 
-    var url = "https://api.github.com/user";
+    const http = new httpClient.HttpClient('typed-rest-client');
 
-    return new Promise((resolve, reject) => {
-        request.get({
-            url : url,
-            headers : {
-                "Authorization": "Token " + token,
-                "User-Agent": "azure-pipelines"
-            }
-        }, function(error, response, body) {
-            if (error) reject(error);
-            let responseJson = JSON.parse(body);
-            resolve(responseJson["login"]);
-        });
+    let res = await http.get("https://api.github.com/user", {
+        "Authorization": "Token " + token,
+        "User-Agent": "azure-pipelines"
```

Comment:
```suggestion
        'User-Agent': 'azure-pipelines'
```
Could it make sense to extract `https://api.github.com/user` and `azure-pipelines` as constants?

Reply:
These strings are only used in one place. It makes no sense to use a variable